src/languages/sqf.js

```diff
@@ -1854,28 +1972,25 @@ export default function(hljs) {
     'setAnimSpeedCoef',
     'setAperture',
     'setApertureNew',
-    'setAPURTD',
     'setArmoryPoints',
     'setAttributes',
     'setAutonomous',
-    'setBatteryChargeRTD',
-    'setBatteryRTD',
     'setBehaviour',
     'setBehaviourStrong',
     'setBleedingRemaining',
     'setBrakesRTD',
-    'setCameraEffect',
     'setCameraInterest',
     'setCamShakeDefParams',
     'setCamShakeParams',
-    'setCamUseTI',
+    'setCamUseTi',
```

Comment:
```suggestion
    'setCamUseTI',
```
https://community.bistudio.com/wiki/setCamUseTI